include/deal.II/hp/fe_values.h

```diff
@@ -80,6 +80,16 @@ namespace hp
       const QCollection<q_dim> &                              q_collection,
       const UpdateFlags                                       update_flags);
 
+    /**
+     * TODO.
+     */
+    FEValuesBase(
+      const MappingCollection<dim, FEValuesType::space_dimension>
+        &mapping_collection,
+      const FECollection<dim, FEValuesType::space_dimension> &fe_collection,
+      const std::vector<QCollection<q_dim>> &                 q_collection,
+      const UpdateFlags                                       update_flags);
```

Comment:
This is the equivalent in the case of hp. The outer vector groups the quadrature indices; the `QCollection` contains the face quadrature rules of once quadrature index.